src/main/java/frc/robot/constants/GameRobotConstants.java

```diff
@@ -86,6 +88,21 @@ public Measure<Velocity<Distance>> getMaxLinearSpeed() {
 
     // #region: --------------- Hardware ---------------------------------------
     // #region: ----- Aimer -----
+    @Override
+    public Tuple2<Rotation2d> getAimerAngleRange() {
+        return new Tuple2<Rotation2d>(Rotation2d.fromDegrees(1), Rotation2d.fromDegrees(45));
+    }
+
+    @Override
+    public Rotation2d getAimerEncoderOffset() {
+        return Rotation2d.fromRadians(2.599);
+    }
+
+    @Override
+    public int getAimerEncoderPort() {
+        return 0;
```

Comment:
Port numbers have the same issue as motor IDs (required uniqueness). Since this is the only one we could go either way.

Reply:
I Moved All CAN Bus ID & roboRIO Port Constants to AbstractConstantsand added uniqueness checks.
Let me know what you think.